sakura_core/window/CEditWnd.cpp

```diff
@@ -4860,6 +4860,19 @@ ECharWidthCacheMode CEditWnd::GetLogfontCacheMode()
 	return CWM_CACHE_SHARE;
 }
 
+/*!
+	@brief 現在のズーム倍率を取得
+	@return 1.0を等倍とするズーム倍率
+*/
+double CEditWnd::GetFontZoom()
+{
+	if( GetDocument()->m_blfCurTemp ){
```

Comment:
対応の必要はないですが👆でnoexceptを付けるならこうなるはずです。

```suggestion
	if( const auto pDocument = GetDocument(); pDocument && pDocument->m_blfCurTemp ){
```

やらんでも良いですが。